x-pack/plugin/eql/qa/common/src/main/resources/test_queries_supported.toml

```diff
@@ -118,3 +118,26 @@ query = "file where 66.0 / serial_event_id == 1"
 [[queries]]
 expected_event_ids = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 46]
 query = "process where serial_event_id + ((1 + 3) * 2 / (3 - 1)) * 2 == 54 or 70 + serial_event_id < 100"
+
+[[queries]]
+query = '''
+sequence
+  [process where serial_event_id = 1]
+  [process where serial_event_id = 2]
+'''
+expected_event_ids  = [1, 2]
+
+[[queries]]
+query = '''
+sequence
```

Comment:
Should we also add another test where the `by` is after `sequence`?
```
sequence
  [process where serial_event_id=1] by unique_pid
  [process where true] by unique_ppid
```

Reply:
We should unblacklist these, too
https://github.com/elastic/elasticsearch/blob/master/x-pack/plugin/eql/qa/common/src/main/resources/test_queries_unsupported.toml#L585-L607

Reply:
Clearly there needs to be more testing in the future and polishing but that's future work. the PR is already large enough - a current issue with the current dataset is that it relies on a custom tie-breaker something not yet implemented.